bip-internalkey.md

```diff
@@ -0,0 +1,68 @@
+| BIP-XXXX   | `OP_INTERNALKEY`
+| :--------- | :---------------
+| Layer      | Consensus (soft fork)
+| Author     | Brandon Black `<freedom@reardencode.com>`, Jeremy Rubin `<j@rubin.io>`
+| Discussion | https://delvingbitcoin.org/t/lnhance-bips-and-implementation/376/9
+| Status     | Draft
+| Type       | Standards Track
+| Created    | 2023-12-22
+| License    | BSD-3-Clause
```

Comment:
Please use the [preamble format](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#bip-header-preamble) specified in BIP2.

Instead of "BIP-XXXX" there should be a "BIP" header. The "Title" header is missing, the "Comments-URI" header is missing, and "Discussion" should be "Post-History".